core/commands/configure_command.rb

```diff
@@ -48,6 +55,29 @@ def execute
 
   private
 
+  def check_dock_credentials(docker_credentials)
+    cmd = "docker login --username #{docker_credentials['username']}" \
+          " --password '#{docker_credentials['password']}' #{docker_credentials['ci-server']}"
+
+    stdin, stdout, stderr = Open3.popen3(cmd)
```

Comment:
You should instead use ShellCommands module, include it into the command and use the provided run_command method from that module.